profiles/base.yaml

```diff
@@ -1190,6 +1190,8 @@ forge_container:
 spell_cycle:
 
 restock_shop:
+# For almanacs that let you pick which skill to use
+almanac_skills:
```

Comment:
What do you think about adding a validator for this? I.e. that everything in the array is a valid skill

Reply:
added a test to validate